adafruit_rfm9x.py

```diff
@@ -519,6 +539,28 @@ def signal_bandwidth(self, val):
             _RH_RF95_REG_1D_MODEM_CONFIG1,
             (self._read_u8(_RH_RF95_REG_1D_MODEM_CONFIG1) & 0x0F) | (bw_id << 4),
         )
+        if val >= 500000:
+            # see Semtech SX1276 errata note 2.3
+            self.auto_ifon = True
```

Comment:
You may want to set the object's `auto_ifon` property in the `__init__`

Reply:
Since the init section sets the bandwidth and spreading factor, auto_ifon will be set there.

Reply:
ok!